cli.js

```diff
@@ -28,6 +29,12 @@ const cli = meow(`
 
 updateNotifier({pkg: cli.pkg}).notify();
 
+const pkg = readPkgUp.sync().pkg;
```

Comment:
Oh yes, you're right. It off course needs the `np` information for `update-notifier`. Seems I need some more coffee :).